torch/csrc/jit/codegen/cuda/partial_split_map.cpp

```diff
@@ -39,7 +39,6 @@ void PartialSplitMap::build(Fusion* fusion) {
 }
 
 Val* PartialSplitMap::getStartOffset(IterDomain* root_domain) const {
-  TORCH_INTERNAL_ASSERT(!root_domain->isKirStmt());
```

Comment:
Does this removal have anything to do with the sorting change?

Reply:
No, likely grabbed this change in this PR by mistake. Would have been done in a later PR anyways.